camera:
  color_topic: "/camera/color"
  gray_topic: "/camera/gray"
  param_topic: "/camera/params"

c922:
  params:
    fx: 627.2
    fy: 630.6
    cx: 295
    cy: 237.1
    k: [0.02, -0.114, 0.004, -0.017, 0.13]
    